cmd/node/app_test.go

```diff
@@ -267,7 +268,7 @@ func txWithUnorderedNonceGenerator(dependencies []int) TestScenario {
 	txsSent := 25
 	setup := func(suite *AppTestSuite, t *testing.T) {
 		for i := 0; i < txsSent; i++ {
-			tx, err := types.GenerateCallTransaction(acc1Signer, dst, uint64(txsSent-i), 10, 1, 1)
+			tx, err := transaction.GenerateCallTransaction(acc1Signer, dst, uint64(txsSent-i), 10, 1, 1)
```

Comment:
I haven't followed the entire test here, but if we didn't add a spawn transaction at nonce == 0, won't this test fail when those are needed?

We should probably make sure that tests without a spawn transaction fail.

Reply:
This `txWithUnorderedNonceGenerator` test is intended to error at https://github.com/spacemeshos/go-spacemesh/blob/develop/cmd/node/app_test.go#L278 because nonces are out of order: from 25 to 0. Perhaps, it would look clearer if the last TX with nonce 0 would be spawn TX, but that wouldn't change the behavior, so IMO it doesn't matter here